src/modules/airdrop/services/airdrop-service.ts

```diff
@@ -104,14 +105,15 @@ export class AirdropService {
     let welcomeTravellerCompleted = false;
 
     if (welcomeTravellerEligible) {
-      const depositCountQuery = await this.depositRepository
+      const depositCountQuery = this.depositRepository
```

Comment:
was this `await` a mistake?

Reply:
not a mistake, but it was not necessary.